Solutions/NonameSecurity/Data Connectors/Connector_RESTAPI_NonameSecurity.json

```diff
@@ -0,0 +1,104 @@
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "NonameAPISecurityAlert_CL",
+            "lastDataReceivedQuery": "NonameAPISecurityAlert_CL\n | summarize Time = max(TimeGenerated)\n | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "NonameAPISecurityAlert_CL\n| summarize LastAlertReceived = max(TimeGenerated)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1,
+        "isPreview": true
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "write": true,
+                    "read": true,
+                    "delete": true
+                }
+            },
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+                "permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key).",
+                "providerDisplayName": "Keys",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "action": true
+                }
+            }
+        ]
+    },
+    "instructionSteps": [
+        {
+            "title": "Configure the Noname Sentinel integration.",
+            "description": "Configure the Sentinel workflow in the Noname integrations settings.  Find documentation at https://docs.nonamesecurity.com",
+            "instructions": [
+                {
+                    "parameters": {
+                        "fillWith": [
+                            "WorkspaceId"
+                        ],
+                        "label": "Workspace ID"
+                    },
+                    "type": "CopyableLabel"
+                },
+                {
+                    "parameters": {
+                        "fillWith": [
+                            "PrimaryKey"
+                        ],
+                        "label": "Primary Key"
+                    },
+                    "type": "CopyableLabel"
+                }
+            ]
+        }
+    ],
+        "metadata": {
+        "id": "f66a9bee-323d-4280-8c3b-e95bc66db83a",
+        "version": "2.0.0",
+        "kind": "dataConnector",
+        "source": {
+            "kind": "solution"
+        },
+        "author": {
+            "name": "Noname Security"
+        },
+        "support": {
+            "tier": "partner",
```

Comment:
@nniryanb, just change the "tier" for data connector. It should be either microsoft , developer or community, thanks.